### PR TITLE
Bug fixes and logging improvements

### PR DESCRIPTION
**Pull request by adjackura**
Package update commands pulled into patch package
Logging cleaned up to be clear on step transitions and record additional metadata

### CODE REVIEW

File: ospatch/updates_linux.go [hunk 1]
```diff
@@ -94,35 +94,35 @@ func systemRebootRequired() (bool, error) {
...
-				opts = append(opts, packages.AptGetUpgradeType(packages.AptGetDistUpgrade))
-			}
+		opts := []AptGetUpgradeOption{AptGetUpgradeRunner(patchRunRunner(r))}
+		switch r.Job.GetPatchConfig().GetApt().GetType() {
+		case osconfigpb.AptSettings_DIST:
+			opts = append(opts, AptGetUpgradeType(AptGetDistUpgrade))
 		}
-		if err := packages.AptGetUpgrade(opts...); err != nil {
+		r.debugf("Installing APT package updates.")
+		if err := retry(3*time.Minute, "installing APT package updates", r.debugf, func() error { return RunAptGetUpgrade(opts...) }); err != nil {
```
Comment (ryanwe) [inline ospatch/updates_linux.go]: Can 3 minutes be a const? Here and below.

File: inventory/packages/apt_deb.go [hunk 2]
```diff
@@ -29,14 +28,11 @@ var (
...
 
-	dpkgQueryArgs         = []string{"-W", "-f", `${Package} ${Architecture} ${Version}\n`}
-	aptGetInstallArgs     = []string{"install", "-y"}
-	aptGetRemoveArgs      = []string{"remove", "-y"}
-	aptGetUpdateArgs      = []string{"update"}
-	aptGetUpgradeArgs     = []string{"upgrade", "-y"}
-	aptGetFullUpgradeArgs = []string{"full-upgrade", "-y"}
-	aptGetDistUpgradeArgs = []string{"dist-upgrade", "-y"}
-	aptGetUpgradableArgs  = []string{"full-upgrade", "--just-print", "-V"}
+	dpkgQueryArgs        = []string{"-W", "-f", `${Package} ${Architecture} ${Version}\n`}
```
[the remark this answers is not in the document]
Comment (adjackura) [reply inventory/packages/apt_deb.go]: Its the format I want the output in:
${Package} ${Architecture} ${Version}\n
${Package} ${Architecture} ${Version}\n
${Package} ${Architecture} ${Version}\n

**Pull request merged**